src/test/kotlin/com/terraformation/backend/db/DatabaseTest.kt

```diff
@@ -378,6 +379,14 @@ abstract class DatabaseTest {
     }
   }
 
+  protected fun getFacilityById(facilityId: FacilityId): FacilitiesRecord {
+    return dslContext
```

Comment:
in case it helps, there's a facilitiesDao 
```
facilitiesDao.fetchOneById(facilityId) ?: throw not found exception
```
might be good to throw an exception in any case (remove !! assumption)